trees/heap.py

```diff
@@ -69,6 +68,9 @@ def __iter__(self):
         while len(self) > 0:
             yield self.pop()
 
+    def __contains__(self, item):
+        return item in self._items
```

Comment:
I'm pretty sure it should be possible to implement this is `O(sqrt(n))`, which is why I'm guessing I didn't implement this. In any case, better to have this implemented and slow, than not implemented at all.